src/main/java/com/ucl/imaginethisserver/Service/ServiceImpl/VoteServiceImpl.java

```diff
@@ -0,0 +1,4 @@
+package com.ucl.imaginethisserver.Service.ServiceImpl;
+
+public class VoteServiceImpl {
+}
```

Comment:
Why this isn't implemented?

Reply:
the voting functions are implemented in the feedbackServiceImpl, is it better to move them here instead?